blocks/api/post.pegjs

```diff
@@ -15,19 +22,35 @@ function maybeJSON( s ) {
 
 }
 
-Document
-  = WP_Block_List
+Block_List
+  = pre:$(!Token .)*
+    ts:(t:Token html:$((!Token .)*) { /** <?php return $t; ?> **/ return [ t, html ] })*
+    post:$(.*)
+  { /** <?php
+    $blocks = [];
```

Comment:
The square bracket array syntax was added in PHP 5.4. We need to use the more-verbose `array()` for support back to PHP 5.2.4

http://php.net/manual/en/language.types.array.php

Reply:
fixed in 4e41437d